src/components/slider/slider.tsx

```diff
@@ -0,0 +1,386 @@
+import { useTheme } from 'native-base';
+import React, { useState, useRef, useEffect } from 'react';
+import { View, Text, PanResponder, Animated, Dimensions } from 'react-native';
+
+import { SliderProps } from '../../types/slider';
+import { Input } from '../inputs';
+
+import { useSliderStyles } from './slider.styles';
+
+enum BubbleHandle {
+  SINGLE = 'single',
+  LEFT = 'left',
+  RIGHT = 'right',
+}
+
+enum SubmitType {
+  MIN = 'min',
+  MAX = 'max',
+}
+
+const Slider = ({
+  value = 0,
+  step = 1,
+  lowerLimit = 0,
+  upperLimit = 100,
+  onValueChange,
+  isRange = false,
+  showEndMarkers = false,
+  internalMarkerStep,
+  showValueInput = false,
+  disabled = false,
+}: SliderProps) => {
+  const { sizes } = useTheme();
+  const screenWidth = Dimensions.get('window').width;
+  const handleSize = Number(sizes['5']);
+  const trackLength = screenWidth * 0.8 - handleSize / 2;
```

Comment:
Slider should take the width of the parent, not the whole window.

That is true in general for a component that it should honor the width of the parent.